tchannel/tornado/connection.py

```diff
@@ -138,6 +137,9 @@ def __init__(self, connection, tchannel=None, direction=None):
         self.tchannel = tchannel
         self._close_cb = None
 
+        self._write_queue = tornado.queues.Queue()
```

Comment:
Consider adding a maxsize to create back pressure on the callers and ensure the queue doesn't grow unreasonably